RecoParticleFlow/PFClusterProducer/python/particleFlowZeroSuppressionECAL_cff.py

```diff
@@ -0,0 +1,7 @@
+import FWCore.ParameterSet.Config as cms
+
+particle_flow_zero_suppression_ECAL = cms.PSet(
+    thresholds = cms.vdouble(
+        0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.250, 0.279, 0.767, 0.674, 0.756, 0.838, 0.8, 1.072, 1.062, 1.002, 1.255, 1.365, 1.092, 1.005, 1.507, 1.121, 1.451, 1.492, 1.746, 1.595, 1.518, 1.626, 2.343, 2.242, 2.102, 2.374, 2.699, 3.288, 3.794, 6.583, 5.455, 9.139, 9.739, 10.0, 10.0, 10.0, 10.0, 10.0, 10.0, 10.0, 0.634, 0.639, 0.686, 0.788, 0.954, 0.82, 0.954, 1.0, 0.953, 0.96, 1.023, 1.033, 0.943, 1.338, 1.125, 1.162, 1.286, 1.249, 1.242, 1.495, 2.809, 2.144, 1.777, 2.064, 2.246, 2.557, 3.587, 4.121, 6.264, 9.926, 10.0, 10.0, 10.0, 10.0, 10.0, 10.0, 10.0, 10.0, 10.0
```

Comment:
please add linebreaks and possibly add some compactification using python repeat pattern ```[0.250]*NNN, 0.279, ... ```
If values can come from a formula, better generate it in-place

This should eventually be a payload.